Sources/Plasma/FeatureLib/pfConditional/plObjectInBoxConditionalObject.cpp

```diff
@@ -67,25 +67,24 @@ bool plObjectInBoxConditionalObject::MsgReceive(plMessage* msg)
         {
             fInside.Append(pActivateMsg->fHitterObj);
             return true;
-        }
-        else
-        if (pActivateMsg->fTriggerType == plActivatorMsg::kVolumeExit)
-        {
-            for (int i = 0; i < fInside.Count(); i++)
+        } else
+            if (pActivateMsg->fTriggerType == plActivatorMsg::kVolumeExit)
```

Comment:
It looks like Visual Studio auto styled this for me. I have no idea how it came up with this crap nor do I remember adding it to the commit. Will fix